tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipsePitSummaryViewTest.java

```diff
@@ -47,16 +47,23 @@ public void navigateWithButtons() throws CoreException {
             // should not change page
             String lastUrl = summaryView.getCurrentBrowserUrl();
             assertThat(summaryView.clickHome(), equalTo(lastUrl));
+            assertThat(summaryView.clickForward(), equalTo(lastUrl));
+            assertThat(summaryView.clickBack(), equalTo(lastUrl));
 
             // coverageReportGenerated needs normal timeout
             SWTBotPreferences.TIMEOUT = timeout;
             runPackageTest(FOO_BAR_PACKAGE, TEST_PROJECT);
             coverageReportGenerated(2, 80, 0);
 
             assertThat(summaryView.getCurrentBrowserUrl(), endsWith(INDEX));
-            assertThat(summaryView.setLink("./"+ FOO_BAR_PACKAGE_RESULT+"/"+FOO_CLASS_RESULT), endsWith(FOO_CLASS_RESULT));
+            assertThat(summaryView.setUrl(summaryView.getCurrentBrowserUrl().replace("/index.html",
```

Comment:
A bit of a hack right now, but even this does not work for the tests.
If I debug it, the browser shows the correct page, but it fails anyway.